testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -166,12 +166,14 @@ class SettingsMenuActivity : BaseActivity() {
         launch {
             try {
                 MiniApp.instance(AppSettings.instance.miniAppSettings).listMiniApp()
+                URL("https://www.example.com$urlParameters").toURI()
+
                 settings.isSettingSaved = true
                 runOnUiThread {
                     settingsProgressDialog.cancel()
                     navigateToPreviousScreen()
                 }
-            } catch (error: MiniAppSdkException) {
+            } catch (error: Exception) {
```

Comment:
We also can utilize it to show meaningful message to the demo app user e.g.

```
if (error.message.toString().contains("Illegal character in fragment"))
   showAlertDialog(this@SettingsMenuActivity, "Invalid url parameter: " + urlParameters)
else
   showAlertDialog(this@SettingsMenuActivity, error.message.toString())
```

Reply:
updated.

Reply:
Ah, I see! 
Ok, according to the new change, is it possible to use `MalformedURLException` here instead of using a generic Exception? Or, we can use a generic `errTitle` in the alert dialog instead of `"URL parameter"`. 
Because not every `Exception` will be under `"URL parameter"`.

Reply:
Not only `MalformedURLException`, there is also `URLException` which could be thrown.

Only parameter check can throw `Exception` in this scope so I used `Exception` here.